edk2toollib/uefi/edk2/parsers/base_parser.py

```diff
@@ -47,6 +48,22 @@ def __init__(self, log="BaseParser"):
     # For include files set the base root path
     #
 
+    def SetEdk2Path(self, pathobj: path_utilities.Edk2Path):
+        """Sets the attribute RootPath.
```

Comment:
better comments here about what is required.  
Reference to deprecated functions as the docs comment wont be much good when those are gone.

Reply:
Good call. I've update the docs with integration instructions and will do the same when announcing the deprecation (but not yet removal) of the two functions in the upcoming release.